public/datasets/titanic.json

```diff
@@ -1,4 +1,5 @@
 {
+  "defaultLabelColumn": "Name",
```

Comment:
Similar to the candy dataset I think the default label column in this case is `Survived`.